addons/project/static/src/js/project_pivot_model.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module **/
+
+import { PivotModel } from "@web/views/pivot/pivot_model";
+
+export class ProjectTaskPivotModel extends PivotModel {
+    /**
+     * @override
+     */
+    _getEmptyGroupLabel(fieldName) {
+        if (fieldName === "project_id") {
+            return this.env._t("Private");
+        } else if (fieldName === "user_ids") {
+            return this.env._t("Unassigned");
```

Comment:
No icons there as you did in the other views? 🧐

Reply:
no cause pivot is a little too serious for that :dagger: